CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Notable changes to the `alacritty_terminal` crate are documented in its
 
 ## 0.15.0-dev
 
+### Changed
+
+- Automatically focus new windows on macOS, even if no Alacritty windows are currently focused
```

Comment:
I think this is a bit long for a changelog message, I'd like to keep it under 80 columns where possible.

```suggestion
- Always focus new windows on macOS
```

I don't think pointing out the delta to the previous behavior is necessary. If people thought it already behaved like this, they can just ignore this changelog.